shared/src/components/exercise/mode.cjsx

```diff
@@ -31,7 +31,7 @@ ExMode = React.createClass
 
   componentDidUpdate: (prevProps) ->
     {mode, focus} = prevProps
-    @setFreeResponseFocusState() if mode is 'free-response' and focus isnt @props.focus
+    @setFreeResponseFocusState() if mode is 'free-response'
```

Comment:
Seems to me, the free-response focuses fine without removing the ` and focus isnt @props.focus` clause?